app/boards/shields/leeloo_micro/leeloo_micro.dtsi

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "leeloo_micro_common.dtsi"
```

Comment:
You should just move the contents of the common file into this one, no need for further abstraction.